cli/task_runner.rs

```diff
@@ -93,13 +93,18 @@ pub struct TaskResult {
 }
 
 pub async fn run_task(
-  opts: RunTaskOptions<'_>,
+  mut opts: RunTaskOptions<'_>,
 ) -> Result<TaskResult, AnyError> {
   let script = get_script_with_args(opts.script, opts.argv);
   let seq_list = deno_task_shell::parser::parse(&script)
     .with_context(|| format!("Error parsing script '{}'.", opts.task_name))?;
   let env_vars =
     prepare_env_vars(opts.env_vars, opts.init_cwd, opts.root_node_modules_dir);
+  if !opts.custom_commands.contains_key("deno") {
+    opts
+      .custom_commands
+      .insert("deno".to_string(), Rc::new(DenoCommand::default()));
+  }
```

Comment:
deno_task_shell no longer handles this resolving `deno` to the current exe, so we handle that here.